### ISSUE TITLE
tBTC dApp: Freezing of the page after rejecting transaction on the step 2

### ISSUE THREAD

**Issue by nikgrego** [keep-network/tbtc-dapp#266]
### Description:
If you reject transaction on the step 2, nothing happens. I assume it's not supposed to be, you need to get any info that user rejected the transaction and redirect him to index page or step 1 page.

### Steps to reproduce the behavior:
1. Go to https://dapp.test.tbtc.network
2. Pass to step 2
3. Reject the transaction
4. See nothing happens

### Expected behavior:
Getting any info that you rejected the transaction and redirection to index page or step 1 page

### Environment details:
OS: OS X El Capitan, 10.11.6 (15G22010)
Web Browser: Google Chrome, Version 83.0.4103.116 (Official Build) (64-bit)  
Wallet: Metamask (Chrome extension), Version 8.0.3

### Screenshots
![Скриншот 2020-07-15 16 50 55](https://user-images.githubusercontent.com/66184067/87553773-ff24b080-c6bb-11ea-92fe-d989abd3a6e4.png)
![Скриншот 2020-07-15 16 51 04](https://user-images.githubusercontent.com/66184067/87553790-021fa100-c6bc-11ea-8ad6-cc9487cd4247.png)

**Comment by Shadowfiend** [comment]
Yeah, in general our handling of Metamask transaction rejections is nonexistent throughout the dApp.
